### PR TITLE
20170929 issue 78

### PR DESCRIPTION
**Pull request by mozmark**
Adds an option to OneCRL2CSV to take bug format data.

### CODE REVIEW

File: oneCRL2CSV/main.go [hunk 1]
```diff
@@ -71,6 +73,12 @@ func main() {
 			panic(err)
 		}
 	} else {
-		oneCRL.LoadRevocationsTxtFromFile(*filePtr, printer)
+		if "revocations.txt" == *fileFormatPtr {
```
Comment (jcjones) [inline oneCRL2CSV/main.go]: Nit: could be a nice switch statement

Comment (mozmark) [reply oneCRL2CSV/main.go]: I wholeheartedly agree.

**Comment by mozmark** [comment]
@jcjones that better? If so, can you approve again, please?

File: oneCRL2CSV/main.go [hunk 2]
```diff
@@ -73,11 +73,14 @@ func main() {
...
-		if "revocations.txt" == *fileFormatPtr {
+		switch *fileFormatPtr {
+		case "revocations.txt":
 			oneCRL.LoadRevocationsTxtFromFile(*filePtr, printer)
-		} else if "bug" == *fileFormatPtr {
+			break;
+		case "bug":
 			oneCRL.LoadRevocationsFromBug(*filePtr, printer)
-		} else {
+			break;
```
Comment (jcjones) [inline oneCRL2CSV/main.go]: Among the cool things about Golang's switch statement is that you don't need to put in `break`s -- the default is to break out, you have to annotate if you want a fall-through.

Comment (mozmark) [reply oneCRL2CSV/main.go]: Every day is a school day!

**Pull request merged**